docs/Components-Header.md

```diff
@@ -0,0 +1,112 @@
+# Header Component
+
+[Back to all components](./Components.md)
+
+The `Header` component is a component that will appear at the top of your screen. It is used to hold navigation buttons and the screen title.
+
+```tsx
+<Header
+  headerTx="header.title"
+  headerText="Header Title"
+  leftIcon="back"
+  rightIcon="bullet"
+  onLeftPress={() => navigation.goBack()}
+  onRightPress={() => Alert.alert("pressed")}
+  style={{ backgroundColor: "purple" }}
+  titleStyle={{ color: "white" }}
+/>
+```
+
+## Props
+
+### `headerTx`
+
+The `headerTx` is an optional prop that is used to lookup the translation for the header title. If this is not set, the `headerText` prop must be present to set the header title. If both are set, the `headerText` value will be used.
+
+```tsx
+  <Header
+    headerTx="header.title"
+    leftIcon="back"
+    onLeftPress={() => navigation.goBack()}
+  />
+```
+
+### `headerText`
+
+The `headerText` is an optional prop that is used to set the header title. If this is not set, the `headerTx` prop must be present to set the title. If both are set, the `headerText` value will be used.
+
+```tsx
+  <Header
+    headerText="Header Title"
+    leftIcon="back"
+    onLeftPress={() => navigation.goBack()}
+  />
+```
+
+### `leftIcon`
+
+The `leftIcon` is an optional prop that is used to set the icon for the left navigation button. Options are 'back', 'bullet', and 'bug'. Custom icons can be created by using the [`Icon` component](./Components-Icon.md#custom-icons).
+
+```tsx
+  <Header
+    leftIcon="back"
+    onLeftPress={() => navigation.goBack()}
+  />
+```
+
+### `rightIcon`
+
+The `rightIcon` is an optional prop that is used to set the icon for the right navigation button. Options are 'bullet', and 'bug'. Custom icons can be created by using the [`Icon` component](./Components-Icon.md#custom-icons).
```

Comment:
Can you explain a bit more how to create a custom icon? Like, do you just create an icon preset and then put the string name in here? Or do you have to provide the <Icon /> JSX directly?